device-types/Cisco/AIR-AP1121G-E-K9.yaml

```
@@ -0,0 +1,13 @@
+---
+manufacturer: Cisco
+model: Aironet 1121G-E
+slug: cisco-air-ap1121g-e-k9
+part_number: AIR-AP1121G-E-K9
+u_height: 0
+is_full_depth: false
+interfaces:
+  - name: WAN
+    type: 100base-tx
```

@DanSheps Do you mean this?
```suggestion
    type: ethernet
```